src/translation/polish.c

```diff
@@ -459,7 +459,7 @@ static translation_string all_strings[] = {
     {TR_BUILDING_WATCHTOWER_DESC, "Łucznicy na krenelażach tej wieży czujnie obserwują, szukając najeźdźców, podczas gdy strażnicy patrolują ulice poniżej." },
     {TR_BUILDING_WINDOW_HOUSE_RECENT_EVENT_POSITIVE, "Mieszkańcy zostali podniesieni na duchu przez ostatnie wydarzenie." },
     {TR_BUILDING_WINDOW_HOUSE_RECENT_EVENT_NEGATIVE, "Mieszkańcy utracili ducha przez ostatnie wydarzenie." },
-    {TR_BUILDING_WINDOW_HOUSE_SENTIMENT_1, "Mieszkańcy nienawidzę Twoje imię."},
+    {TR_BUILDING_WINDOW_HOUSE_SENTIMENT_1, "Mieszkańcy nienawidzą Twoje imię."},
```

Comment:
```suggestion
    {TR_BUILDING_WINDOW_HOUSE_SENTIMENT_1, "Mieszkańcy wzdrygają się na sam dźwięk Twojego imienia!."},
```
Jeśli bardzo chcemy zachować oryginalną treść, to trzeba zmienić przypadek:
'Mieszkańcy nienawidzą Twojego imienia' 
Ale chyba nikt tak nie mówi, a 'wzdrygiwać się na dźwięk imienia' to stosunkowo popularna fraza, występująca m.in. w polskim tłumaczeniu książek o Harrym Potterze